### ISSUE TITLE
(Minor) Rephrase 'nonzero entries' in `display` of `SparseMatrix`

### ISSUE THREAD

**Issue by colinfang** [JuliaLang/julia#19119]
I would like to suggest we replace `nonzero` with `stored` so that it brings less confusion.

Also, there are a lot of places in docs and in code we use the phrase 'structural nonzero values'. IMHO, I find it less readable than a simple 'stored values'.

``` julia
x = speye(4)
x[1, 1] = 0
display(x)

4×4 sparse matrix with 4 Float64 nonzero entries:
    [1, 1]  =  0.0
    [2, 2]  =  1.0
    [3, 3]  =  1.0
    [4, 4]  =  1.0
```

**Comment by akaysh** [comment]
Hi, I am a beginner and want to start contributing to Julia. If this is doable by a beginner, can I take up this issue? 
And can anyone guide me on how to approach the problem and what needs to be done?
Thanks

**Comment by StefanKarpinski** [comment]
Go for it. This should be pretty straightforward.

**Comment by fredrikekre** [comment]
Fixed by #19483 and improved by #20488
